packages/forma-36-website/src/pages/guidelines/copy/grammar/punctuation/index.mdx

```diff
@@ -0,0 +1,40 @@
+# Punctuation
+
+## Use periods consistently
+
+Keep UX copy scannable by eliminating unnecessary periods.
+Do not use periods on:
+
+- buttons
+- labels
+- tooltips
+- bulleted lists
```

Comment:
This rule doesn't always make sense, like if the list item is a fully-formed sentence.